index.js

```diff
@@ -26,8 +26,17 @@ const RouterCtx = createContext({});
 const buildRouter = ({
   hook = locationHook,
   base = "",
-  matcher = makeMatcher(),
-} = {}) => ({ hook, base, matcher });
+  matcher,
+  parent,
+} = {}) => ({
+  hook,
+  relbase: base,
```

Comment:
I'm not a big fan of the naming `relbase` here. Is it a contraction of `relative` + `base`? In this case I'd at least make it camelcase (`relBase`). I'd even go for `relativeBase` but that adds a few bytes which is a shame for so little gain.

Reply:
I could do camelcase for sure... yes, "relative" + "base" was what I was going for.

Reply:
Pondered this question some more... I think I will switch to `ownBase`! Short and sweet and completely obvious semantics.